tests/test_backend_netwitness.py

```diff
@@ -224,14 +224,17 @@ def test_netwitness_contains_modifier_with_list(netwitness_backend: NetWitnessBa
             detection:
                 sel:
                     fieldA|contains:
+                        # This is a comment
```

Comment:
Consider splitting this into multiple tests:
- `test_netwitness_contains_with_inline_comments`
- `test_netwitness_contains_with_single_value`
- `test_netwitness_contains_with_list`

Reply:
I would like to keep it in a single file if that's okay for you :-)